packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -413,6 +413,29 @@ describe('<Autocomplete />', () => {
         expect(getAllByRole('button', { hidden: false })).to.have.lengthOf(5);
       }
     });
+
+    it('listbox stays open after opening', async () => {
```

Comment:
This test passes in the `next` HEAD branch, so it isn't testing correctly what's needed. It should pass in this branch but fail in the `next` branch.

Reply:
I got it, I'll modify it, soon!!